actors/providers/redis/src/main/java/com/ea/orbit/actors/providers/redis/RedisStorageProvider.java

```diff
@@ -60,14 +56,13 @@ public RedisStorageProvider() {
 				.withFieldVisibility(JsonAutoDetect.Visibility.ANY)
 				.withGetterVisibility(JsonAutoDetect.Visibility.NONE)
 				.withSetterVisibility(JsonAutoDetect.Visibility.NONE)
-				.withSetterVisibility(JsonAutoDetect.Visibility.NONE)
 				.withCreatorVisibility(JsonAutoDetect.Visibility.NONE));
 		redis = new Jedis(host, port);
 		return Task.done();
 	}
 
 	private String asKey(final ActorReference reference) {
-		String classname = ActorReference.getInterfaceClass(reference).getName();
+		String classname = ActorReference.getInterfaceClass(reference).getSimpleName();
```

Comment:
@quilombodigital let me know if this was intentional and I'll revert the revert :)